README.md

```diff
@@ -8,15 +8,24 @@ Each student has different learning goals and each project is different so this
 
 Some mentorship pairs may be ready immediately to begin planning a final project, others may not advance that far through the program – you can use your first session to find out where in the curriculum to start.
 
-## How to navigate this repository
+---
+
+## Navigate by Phases
+### [Phase 1: Digital Literacy](digital-literacy/phase-1.md)
+### [Phase 2: Web Fundamentals](webdev/phase-2.md)
+### [Phase 3: Intro to Programming](webdev/phase-3.md)
+
+---
+
+## Emergent Work Resources
 
 As of 2022, Emergent Works will focus on:
 
-[Web Development](webdev)
+[Web Development](webdev/README.md)
```

Comment:
I think we probably need to work this content into the Phase 1,2,3 write-ups.   As it is here, it’s kind of a “head-scratcher” as to how it relates to the phases.  (Probably just need to figure out what part of webdev/README.md needs to be brought into phase 2 write-up)

Reply:
As of my changes, this is pretty much an unreachable page (just FYI) (pretty sure)